app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, foreign_key: :author_id
   belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count + 1)
+  end
```

Comment:
- [ ] It would be better if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database, then make the `update_comments_counter` method private :blush: